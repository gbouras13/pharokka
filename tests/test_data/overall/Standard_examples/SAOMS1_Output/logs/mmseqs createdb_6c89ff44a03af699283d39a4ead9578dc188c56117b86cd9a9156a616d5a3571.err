Command line: mmseqs createdb SAOMS1_Output/phanotate_aas_tmp.fasta SAOMS1_Output/CARD_target_dir/target_seqs
