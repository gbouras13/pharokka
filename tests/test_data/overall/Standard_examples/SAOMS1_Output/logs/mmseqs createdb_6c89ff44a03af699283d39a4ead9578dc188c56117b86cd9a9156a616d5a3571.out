createdb SAOMS1_Output/phanotate_aas_tmp.fasta SAOMS1_Output/CARD_target_dir/target_seqs 

MMseqs Version:       	13.45111
Database type         	0
Shuffle input database	true
Createdb mode         	0
Write lookup file     	1
Offset of numeric ids 	0
Compressed            	0
Verbosity             	3

Converting sequences
[
Time for merging to target_seqs_h: 0h 0m 0s 13ms
Time for merging to target_seqs: 0h 0m 0s 12ms
Database type: Aminoacid
Time for processing: 0h 0m 0s 52ms
